tests/examples/test_building_deploying_multi_stage_RecSys.py

```diff
@@ -45,18 +44,22 @@ def test_func(tb1):
         tb2.execute_cell(list(range(0, NUM_OF_CELLS - 3)))
         top_k = tb2.ref("top_k")
         outputs = tb2.ref("outputs")
-        request = tb2.ref('request')
+        request = tb2.ref("request")
         assert outputs[0] == "ordered_ids"
         tb2.inject(
             """
+            import shutil
             from merlin.models.loader.tf_utils import configure_tensorflow
             configure_tensorflow()
             from merlin.systems.triton.utils import run_ensemble_on_tritonserver
-            response = run_ensemble_on_tritonserver('/tmp/examples/poc_ensemble', outputs, request, 'ensemble_model')
-            
+            response = run_ensemble_on_tritonserver('/tmp/examples/poc_ensemble',
+                                                    outputs,
+                                                    request,
+                                                    'ensemble_model')
+            response = [x.tolist()[0] for x in response['ordered_ids']]
+            shutil.rmtree('/tmp/examples/', ignore_errors=True)
```

Comment:
super minor nitpick: the code formatting in this string block doesn't match the rest of our repos (black - so should be using double quotes for strings instead of single quotes). I know its a string thats getting injected into the notebook - so the black lint check isn't pickign this up - but how about something like :

```suggestion
            response = run_ensemble_on_tritonserver(
                "/tmp/examples/poc_ensemble", outputs, request, "ensemble_model"
            )
            response = [x.tolist()[0] for x in response["ordered_ids"]]
            shutil.rmtree("/tmp/examples/", ignore_errors=True)
```